cypress/support/commands.js

```diff
@@ -0,0 +1,39 @@
+function normalizeCoords(coords) {
+  // Note: Cypress seems to check against falsiness (thus 0 values are
+  //  considered as not supplied, and position defualts to "center").
+  //  Until fixed, default to `1`
+  // See https://github.com/cypress-io/cypress/issues/2338
+  return {
+    x: coords.x || 1,
+    y: coords.y || 1
+  };
+}
+
+Cypress.Commands.add("pickTool", { prevSubject: false }, tool =>
+  cy.contains("label", tool).click()
```

Comment:
For this label is too generic. We can use a get by [data-testid](https://docs.cypress.io/api/commands/get.html#Find-5-elements-with-the-given-data-attribute).

Reply:
It is, but since it's its own command, all it takes is to update one selector. You *can* add `data-testid`, if you want, though.